scripts/azureml-assets/azureml/assets/release/model_publish_utils/utils.py

```diff
@@ -49,33 +42,40 @@ def _run(self, cmd, cwd: Path = "./") -> int:
             logger.print(f"Successfully executed! Output: \n{result.stdout}")
         return result.returncode
 
-    def _download_git_model(self) -> bool:
+    def _download_git_model(model_url, model_dir) -> bool:
```

Comment:
(nit) Please add docs for updated params

Reply:
Done